entrypoint.sh

```diff
@@ -1,19 +1,77 @@
-#!/bin/sh
-set -e # Exit immediately if a command exits with a non-zero status
+#!/bin/bash
+set -eu # Increase bash strictness
+set -o pipefail
```

Comment:
Do we need this pipefail?
If we unset pipefail, you can use `PIPESTATUS` bash feature to get each command in pipe instead of capturing `remark_lint_output`.

Reply:
@haya14busa I would use pipefail with proper pipefailure.

Reply:
@haya14busa I tried using PIPESTATUS previously, but I run into problem when the Reviewdog help dialogue is returned due to incorrect (user supplied) Reviewdog flags (see my [stackoverflow issue](https://stackoverflow.com/questions/65485359/redirecting-stderr-changes-exit-codes-of-piped-command)). This was however related to me wanting to access unchanged exit code of black while it was piped into Reviewdog. As I now first store the black output in a seperate variable the pipefail can indeed be replaced with using PIPESTATUS. @limakzi Why do you suggest using pipefail over PIPESTATUS?

Reply:
@haya14busa and @limakzi I removed the error catching behavoir after all since it is not needed if we don't do the formatting.